virtool/account/api.py

```diff
@@ -182,7 +182,7 @@ async def get_api_key(req: aiohttp.web.Request) -> aiohttp.web.Response:
     document = await db.keys.find_one({"id": key_id, "user.id": user_id}, API_KEY_PROJECTION)
 
     if document is None:
-        return not_found()
+        raise HTTPNotFound(text="Not found")
```

Comment:
I think we should subclass the aiohttp `HTTPNotFound` exception to make our own `HTTPNotFound` that sets the default value for `text`.

Reply:
Sounds good!